pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/crd/kustomization.go

```diff
@@ -68,9 +68,9 @@ func (f *Kustomization) GetMarkers() []machinery.Marker {
 const (
 	resourceCodeFragment = `- bases/%s_%s.yaml
```

Comment:
Thank you for the contribution

The kustomize/v1 is only used in the deprecated layouts
It is deprecated as well.

So, we should only fix it in the kustomize/v2

Could you please remove the change from this file?